backend/src/openarchiefbeheer/utils/tests/gherkin.py

```diff
@@ -619,10 +619,23 @@ async def user_fills_form_field(self, page, label, value, role=None, index=0):
             elements = await locator.all()
             await elements[index].fill(value)
 
-        async def user_filters_zaken(self, page, name, value):
+        async def user_filters_zaken_on_text(self, page, name, value):
```

Comment:
Wouldn't it be nice to hide the specifics of the field from the user, just like `user_fills_form_field` (which me may be able to use)?

Reply:
Will be refactored in https://github.com/maykinmedia/open-archiefbeheer/issues/727.
Discussed in the office, merging for now.